ui/src/submissions/literature/components/__tests__/__snapshots__/DataImporter.test.jsx.snap

```diff
@@ -20,7 +20,105 @@ exports[`DataImporter renders with error with a message 1`] = `
     <Col>
       <Alert
         closable={true}
-        message="error"
+        message={
+          <React.Fragment>
+            error
+          </React.Fragment>
+        }
+        showIcon={true}
+        type="error"
+      />
+    </Col>
+  </Row>
+  <FormItem
+    hasFeedback={false}
+    label="From arXiv or DOI"
+    labelCol={
+      Object {
+        "span": 5,
+      }
+    }
+    wrapperCol={
+      Object {
+        "span": 19,
+      }
+    }
+  >
+    <Input
+      data-test-id="import-input"
+      disabled={false}
+      onChange={[Function]}
+      onPressEnter={[Function]}
+      placeholder="hep-th/9711200 or 1207.7235 or arXiv:1001.4538 or 10.1086/305772 or doi:10.1086/305772"
+      type="text"
+    />
+  </FormItem>
+  <Row
+    align="middle"
+    gutter={0}
+    justify="space-between"
+    type="flex"
+  >
+    <Col>
+      <LinkLikeButton
+        dataTestId="skip-import-button"
+        onClick={[MockFunction]}
+      >
+        Skip and fill the form manually
+      </LinkLikeButton>
+    </Col>
+    <Col>
+      <Button
+        block={false}
+        data-test-id="import-button"
+        ghost={false}
+        htmlType="button"
+        loading={false}
+        onClick={[Function]}
+        type="primary"
+      >
+        Import
+      </Button>
+    </Col>
+  </Row>
+</div>
+`;
+
+exports[`DataImporter renders with error with message and recid 1`] = `
+<div>
+  <Row
+    className="mb3"
+    gutter={0}
+  >
+    <Col>
+      <Alert
+        message="Fill in the field to automatically import more data from arXiv or DOI"
+        type="info"
+      />
+    </Col>
+  </Row>
+  <Row
+    className="mb3"
+    gutter={0}
+  >
+    <Col>
+      <Alert
+        closable={true}
+        message={
+          <React.Fragment>
+            error
+            <React.Fragment>
+              . 
+               
+              <Link
+                replace={false}
+                to="/literature/undefined"
```

Comment:
something seems be wrong, because you're accessing `error.recid` instead of `error.get('recid')`